README.md

```diff
@@ -119,6 +131,48 @@ With just a click of a button, you can navigate backwards and forwards through t
   <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/revision-navigation.gif" alt="Revision Navigation" />
 </figure>
 
+# Accelerate PR Reviews
+
+PR Reviews can take time and require context switching between GitHub, email, and your IDE. Launchpad consolidates PRs into an actionable hub where you can manage them all in one place. Seamlessly prioritize tasks and identify bottlenecks - all within VS Code when you integrate your Github or host provider. Work on multiple branches simultaneously with Worktrees — hotfixes, new features, or experiments without disrupting your main workspace.
+
+## Launchpad `Pro`
+
+Launchpad brings all of your GitHub pull requests into a unified, actionable list to better track work in progress, pending work, reviews, and more. Stay focused and take action on the most important items to keep your team unblocked. [Learn more](https://gitkraken.com/solutions/launchpad?utm_source=gitlens-extension&utm_medium=in-app-links)
+
+<figure align="center">
+  <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/launchpad.png" alt="Launchpad" />
+</figure>
+
+## Worktrees `Pro`
+
+Efficiently multitask by minimizing the context switching between branches, allowing you to easily work on different branches of a repository simultaneously.
+
+Avoid interrupting your work in progress when needing to review a pull request. Simply create a new worktree and open it in a new VS Code window, all without impacting your other work.
+
+<figure align="center">
+  <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/worktrees.png" alt="Worktrees view" />
+</figure>
+
+
+# Streamline Collaboration
+
+GitLens is more than a solo dev tool - it's built to improve developver collaboration. It's not always easy to share code without adding commits and branches that end up bloating your repo. With the use of Cloud Patches and Code Suggest, GitLens offers the ability to easily share code or suggest code changes to any file in the repository without needing to commit or push it to remote. 
```

Comment:
Call back to the section earlier with some simplification of the description of Cloud Patches and Code Suggest since we're getting ready to talk about them next.

```suggestion
GitLens isn’t just for solo developers—it’s designed to enhance team collaboration. Sharing code can be tricky without adding noise to your repository with extra commits or branches. GitLens simplifies this with Cloud Patches and Code Suggest, letting you share or propose changes to any file in the repository without committing or pushing to a remote.
```